packages/vsx-registry/src/browser/style/index.css

```diff
@@ -49,6 +49,9 @@
 .theia-vsx-extension {
     display: flex;
     flex-direction: row;
+    padding: 0 0 0 calc(var(--theia-ui-padding)*4/3);
```

Comment:
I've removed a padding on purpose here: https://github.com/kaiyue0329/theia/blob/d3bb302e470901cc18276486aa99c1a88af83cf3/packages/vsx-registry/src/browser/vsx-extensions-widget.ts#L77

Please remove this method instead and revert styles here.